bitnami/kafka/values.yaml

```diff
@@ -802,3 +802,90 @@ externalZookeeper:
+## log4j.appender.requestAppender.DatePattern='.'yyyy-MM-dd-HH
+## log4j.appender.requestAppender.File=${kafka.logs.dir}/kafka-request.log
+## log4j.appender.requestAppender.layout=org.apache.log4j.PatternLayout
+## log4j.appender.requestAppender.layout.ConversionPattern=[%d] %p %m (%c)%n
+##
+## log4j.appender.cleanerAppender=org.apache.log4j.DailyRollingFileAppender
+## log4j.appender.cleanerAppender.DatePattern='.'yyyy-MM-dd-HH
+## log4j.appender.cleanerAppender.File=${kafka.logs.dir}/log-cleaner.log
+## log4j.appender.cleanerAppender.layout=org.apache.log4j.PatternLayout
+## log4j.appender.cleanerAppender.layout.ConversionPattern=[%d] %p %m (%c)%n
+##
+## log4j.appender.controllerAppender=org.apache.log4j.DailyRollingFileAppender
+## log4j.appender.controllerAppender.DatePattern='.'yyyy-MM-dd-HH
+## log4j.appender.controllerAppender.File=${kafka.logs.dir}/controller.log
+## log4j.appender.controllerAppender.layout=org.apache.log4j.PatternLayout
+## log4j.appender.controllerAppender.layout.ConversionPattern=[%d] %p %m (%c)%n
+##
+## log4j.appender.authorizerAppender=org.apache.log4j.DailyRollingFileAppender
+## log4j.appender.authorizerAppender.DatePattern='.'yyyy-MM-dd-HH
+## log4j.appender.authorizerAppender.File=${kafka.logs.dir}/kafka-authorizer.log
+## log4j.appender.authorizerAppender.layout=org.apache.log4j.PatternLayout
+## log4j.appender.authorizerAppender.layout.ConversionPattern=[%d] %p %m (%c)%n
+##
+## # Change the line below to adjust ZK client logging
+## log4j.logger.org.apache.zookeeper=INFO
+##
+## # Change the two lines below to adjust the general broker logging level (output to server.log and stdout)
+## log4j.logger.kafka=INFO
+## log4j.logger.org.apache.kafka=INFO
+##
+## # Change to DEBUG or TRACE to enable request logging
+## log4j.logger.kafka.request.logger=WARN, requestAppender
+## log4j.additivity.kafka.request.logger=false
+##
+## # Uncomment the lines below and change log4j.logger.kafka.network.RequestChannel$ to TRACE for additional output
+## # related to the handling of requests
+## #log4j.logger.kafka.network.Processor=TRACE, requestAppender
+## #log4j.logger.kafka.server.KafkaApis=TRACE, requestAppender
+## #log4j.additivity.kafka.server.KafkaApis=false
+## log4j.logger.kafka.network.RequestChannel$=WARN, requestAppender
+## log4j.additivity.kafka.network.RequestChannel$=false
+##
+## log4j.logger.kafka.controller=TRACE, controllerAppender
+## log4j.additivity.kafka.controller=false
+##
+## log4j.logger.kafka.log.LogCleaner=INFO, cleanerAppender
+## log4j.additivity.kafka.log.LogCleaner=false
+##
+## log4j.logger.state.change.logger=TRACE, stateChangeAppender
+## log4j.additivity.state.change.logger=false
+##
+## # Access denials are logged at INFO level, change to DEBUG to also log allowed accesses
+## log4j.logger.kafka.authorizer.logger=INFO, authorizerAppender
+## log4j.additivity.kafka.authorizer.logger=false
+##
+##
+##
+# log4j:
```

Comment:
For `server.properties` we also support reusing an existing configmap. See:

```
## Kafka Configuration
## Specify content for server.properties
## The server.properties is auto-generated based on other parameters when this paremeter is not specified
##
# config:

## ConfigMap with Kafka Configuration
## NOTE: This will override config
##
# existingConfigmap:
```
We should also support this for `log4j`